src/components/Button/index.tsx

```diff
@@ -237,6 +240,7 @@ function Button(
         shouldShowRightIcon = false,
 
         id = '',
+        testID = undefined,
```

Comment:
Unspecified properties are `undefined` by default? Why do we need to explicitly assign `undefined` here?

Reply:
Hmm, there are 2 other places a few lines above where we do the same. But now I'm wondering why we reassign other props like `onLayout` and `id` as they could also be `undefined` :thinking:

Reply:
Ah, I see
Then it's fine I believe 👍